### ISSUE TITLE
[False positive] ::class result and the class qualified name are not identical (case mismatch).

### ISSUE THREAD

**Issue by gnumoksha** [kalessil/phpinspectionsea#1474]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.2 |
| Language level | PHP 7.2           |

Current behaviour
--

The following class will throw the message "[EA] ::class result and the class qualified name are not identical (case mismatch)."

```php
declare(strict_types=1);

namespace Potato\Application\Service;

use Potato\Application\Service\Foo\Bar\Baz as TBaz;
use Potato\Foo\Bar\Baz as PBaz;

class SomeBuilder
{
    public function build(int $number) : object
    {
        $providers = [
            1 => PBaz\Test01::class,
            2 => TBaz\Test02::class,
        ];

        $class = $providers[$number];

        return new $class();
    }
}

namespace Potato\Foo\Bar\Baz {

    class Test01
    {
    }
}

namespace Potato\Application\Service\Foo\Bar\Baz {

    class Test02
    {
    }
}
```

![Screenshot from 2019-12-23 14-14-43](https://user-images.githubusercontent.com/696797/71371056-a27e6e80-258e-11ea-87b8-43b993612ed4.png)

This only occurs if the aliased namespace is part of the current class's namespace.

Expected behaviour
--
No error is shown.

Environment details
---

**Comment by kalessil** [comment]
Will check, thank you for reporting @gnumoksha.

**Comment by kalessil** [comment]
Fixed!
